src/test/java/seedu/address/testutil/LessonBuilder.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.testutil;
```

Comment:
I like you you used the ClassBuilder style to create dummy data for test cases, I will be sure to use it too for my future test cases.